src/Database/TenantDatabaseManagers/SQLiteDatabaseManager.php

```diff
@@ -10,10 +10,12 @@
 
 class SQLiteDatabaseManager implements TenantDatabaseManager
 {
+    public static string|null $path = null;
```

Comment:
Is this essentially a new feature? Customizing the path used by this DB manager.

Reply:
> Is this essentially a new feature?

Yes, it is. So completing the test is first adding the ability to change the path.